### PR TITLE
Documneted rest APIs offered by CGRateS

### PR DESCRIPTION
**Pull request by basitengr**
These documents will merge with main stream documentation once be fully completed.

### CODE REVIEW

File: docs/readthedocs/cdrManagement.rst [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    		"ExportedCGRIDs": null,
+    		"UnexportedCGRIDs": null
+    	},
+    	"error": null
+    }
+
+
+Or fetch CDRs from mongodb
+
+4.2 List all CDRs
```
Comment (danbogos) [inline docs/readthedocs/cdrManagement.rst]: We don't encurage any direct interaction with the database. Please use this API if you want to exemplify CDR queries:
https://godoc.org/github.com/cgrates/cgrates/apier/v2#ApierV2.GetCdrs

Thanks!

**Comment by danbogos** [comment]
Please correct the CDR queries, other than this it should be OK

**Comment by danbogos** [comment]
hey @basitengr 
When do you think you have time to do the changes?
Asking this since I am trying to avoid keeping the PR for too long time.

Ta,
DanB

**Comment by basitengr** [comment]
Hi Dan,

I will do it tonight. What i have plan is to remove mongodb query part. 
We can provide more filters examples with an update.

**Comment by danbogos** [comment]
Thanks!

**Pull request merged**